web/src/main/kotlin/no/nav/su/se/bakover/web/routes/revurdering/BeregnOgSimulerRevurderingRoute.kt

```diff
@@ -61,7 +63,8 @@ internal fun Route.beregnOgSimulerRevurdering(
                                 ).mapLeft {
                                     it.tilResultat()
                                 }.map { revurdering ->
-                                    call.audit("Opprettet en ny revurdering beregning og simulering på sak med id $sakId")
+                                    call.sikkerlogg("Opprettet en ny revurdering beregning og simulering på sak med id $sakId")
```

Comment:
"Opprettet" henger ikke sammen med Action.UPDATE

Reply:
Hm, ser ut som denne metoden egentlig barer beregner og simulerer (på nytt) for en revurdering? Isåfall er UPDATE OK for audit-kallet, men sikkerLogg-kallet burde si noe om at den beregner/simulerer på en revurdering. Usikker på hvorfor den bare refererer til saken..

```suggestion
                                    call.sikkerlogg("Beregnet og simulert revurdering $revurdering.id på $sakId")
```

Evt. bare sløyfe sakId når vi har revurdering.id.